releasenotes/notes/1.3/backend-sampler-v2-level1-dc13af460cd38454.yaml

```
@@ -1,15 +1,15 @@
 ---
 features:
```

This should be `features_primitives`

```suggestion
features_primitives:
```